dev-client/src/screens/AppScaffold.tsx

```diff
@@ -130,8 +102,16 @@ export default function AppScaffold() {
   // using theme hook here because react-navigation doesn't take nativebase utility props
   const {colors} = useTheme();
 
-  const filterLogin = (name: ScreenName) =>
-    (currentUser === null) === (name === 'LOGIN');
```

Comment:
note: i was poking around here trying to improve navigation performance by trying not to pass different props into `RootStack.Navigator`. hasn't fixed everything but i've noticed some improvements i think? and i think this is more idiomatic. there is associated logic for this change in the `LoginScreen` (if the LoginScreen renders and there is a current user, navigates to the home screen)